machine/v1beta1/types_gcpprovider.go

```diff
@@ -134,6 +147,15 @@ type GCPKMSKeyReference struct {
 	Location string `json:"location"`
 }
 
+// GCPAcceleratorConfig describes type and count of accelerator cards attached to the instance on GCP.
+type GCPAcceleratorConfig struct {
+	// AcceleratorCount is number of AcceleratorType accelerators (GPUs) to be attached to an instance
+	AcceleratorCount int64 `json:"acceleratorCount"`
+	// AcceleratorType is the type of accelerator (GPU) to be attached to an instance.
+	// Supported accelerator types are: nvidia-tesla-k80, nvidia-tesla-p100, nvidia-tesla-v100, nvidia-tesla-a100, nvidia-tesla-p4, nvidia-tesla-t4
```

Comment:
need validation tags.

Reply:
Should we make these typed constants as I suggested above?

Do we need validation tags when we this API is only used as a raw extension?

Reply:
If we make these into the typed constants, we will have to update this in case gcp changes supported cards. 
Also this line was supposed to be more of comment of which GPUs does gcp allow. If we were to make this into constants accepted by `AcceleraorType` field, I will remove nvidia-tesla-a100 type since it cant be specified in this field. This type is supported only in machine types with already attached gpus.